### PR TITLE
Kraken: Fill stop_area instead of stop_point

### PR DESCRIPTION
**Pull request by l-vincent-l**
For the from of a first section if the origin is a stop_area
The same thing is done for the to of a last section

### CODE REVIEW

File: source/routing/raptor_api.cpp [hunk 1]
```diff
@@ -139,43 +140,62 @@ pbnavitia::Response make_pathes(const std::vector<navitia::routing::Path>& paths
...
-                            departure_ptime = p_deptime;
-                        // L'heure d'arrivée au dernier stop point
-                        arrival_ptime = p_arrtime;
-                        if(i>0) {
-                            const auto & previous_coord = item.stop_points[i-1]->coord;
-                            const auto & current_coord = item.stop_points[i]->coord;
-                            length += previous_coord.distance_to(current_coord);
-                        }
+                    if (vj->has_boarding() || vj->has_landing()) {
+                        continue;
```
Comment (antoine-de) [inline source/routing/raptor_api.cpp:144]: \o/

**Comment by l-vincent-l** [comment]
Is that OK with you if i nest a stop point in the stop area?

**Comment by koch-t** [comment]
I could work with that but i do dislike the inconsistency of it.

**Comment by l-vincent-l** [comment]
It was made for consistency purpose, we want you to have in the form of the first section (and in the "to" of the last section) the object you asked in the the from (respectively the "to") parameter of your request.

**Pull request merged**